### DISCUSSION TITLE
Using docker-compose, make the wiki public

### DISCUSSION THREAD

**Discussion by lucasballr** [requarks/wiki#6100] [Help / Questions] [answered]
I've launched the server using the provided docker-compose.yml from the docs. My problem is, I want these docs to be accessible without login. I saw that normally you would have to change the config.yml to "public: true" but I don't see a way to do that with the docker-compose file.

**Comment by NGPixel** [comment, marked as answer]
There's no "public: true" config. You need to change the Guests group permissions. See https://docs.requarks.io/groups

However, the default config is for the Guests group to have a read access... Which version are you using?

**Reply by lucasballr** [reply]
I'm using version 2.5.295. I've set the guest group to have read access, but there's still no way of accessing the wiki without logging in to some account. I can't even log in to the "guest" user account since the password is hidden.

**Reply by NGPixel** [reply]
That's not how it works. The guest group is what you see when not logged in. It's not an account you can login to.

Post a screenshot of the Permissions and Page Rules tabs of the Guest group?

**Reply by lucasballr** [reply]
Here you go
![image](https://user-images.githubusercontent.com/63262961/215406815-a05e05f9-cda5-4f34-906f-9b1c9b036e80.png)
![image](https://user-images.githubusercontent.com/63262961/215406922-9fd9e8f6-9746-4472-bec0-ecf2805723a8.png)
You can try it yourself: https://wiki.osusec.org

**Reply by NGPixel** [reply]
Either set the path to /home instead of / or use "Path Starts With".

**Reply by lucasballr** [reply]
Okay interesting. I didn't realize the "/" was not the default directory. Thanks for the help.
